.github/workflows/preview.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         with:
           branch: gh-pages
           folder: demo/dist
-          target-folder: ${{ github.sha }}
+          target-folder: pull-requests/${{ github.sha }}
```

Comment:
`github.event.pull_request.number`?
https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types#pullrequestevent

Reply:
Yeah I was going to, but then we only get 1 preview url per PR.

What would you think about:

`pull-requests/${{ github.sha }}/${{ github.event.pull_request.number }}`

or:

`commits/${{ github.sha }}`

Reply:
I've gone with `commits` for now, as `pull-requests` didn't really make sense.

Reply:
1 preview URL is fine isn't it?
Why do you need multiple different previews live for a PR? The job is `on: pull_request` and so it's the Pull Request we are concerned with previewing.
I imagine it would also make cleaning up easier as you could see which PR is being merged and clean up that preview.

Reply:
I dunno. I think it's usful to be able to follow the comment history in a PR and see a preview of what people were discussing "at that point in time".

But I've changed it [now](https://github.com/votemike/react-accordion/pull/17/commits/20c49e472472fe819a7b1b71f4bd6e15335a1f40).